shared/video-sdk/get-started/get-started-sdk/project-implementation/unity.mdx

```diff
@@ -308,9 +296,9 @@ To implement this logic, take the following steps:
     ```
     </ProductWrapper>
 
-1. **Join a channel to start <Vpl k="NAME" />**
+1. **Join a channel to start <Vpd k="PRODUCT" />**
 
-    When the local user clicks the `Join` button, you call `Join()`. This method securely connects the local user to a channel using the authentication token. Add the following lines after `InitEventHandler`:
+    When the local user clicks the `Join` button, you call `Join()`. This method securely connects the local user to a channel using the authentication token. In your script file, add the following lines after the `InitEventHandler` method:
```

Comment:
after `InitEventHandler`:

Global commet. Please don't write the `whatever` method. It is in code, we know it is a method. 

after `methodname`: 

is enough